introduction.md

```diff
@@ -6,27 +6,101 @@ title: Introduction
+* **Audit**: Detailed audit trail with per-field granularity and `diff` feature for templates.
+
+**Remote execution (plugin)**
+* **Job invocations**: Running arbitrary commands or scripts on remote hosts using different providers, such as SSH or Ansible. This includes scheduling future runs, recurring execution, concurrency control, watching the progress and output live.
+
+**Puppet integration**
+* **Puppet classes**: Ability to import and parse Puppet source code base and recognize class parameters for deep mapping integration through the application.
+* **Puppet CA**: Integration with puppet CA for automatic, semi-automatic or fully automatic client cert sign process.
+* **Puppet ENC**: Puppet node classifier (source of input) for Puppet Master.
+* **Configuration reports**: Inventory of reports from configuration management systems with diff feature and runtime statistics and graphs.
+
+**Ansible integration (plugin)**
+* **Ansible roles**: Ability to import and parse Ansible source code for deeper integration. In combination with remote execution, provides configuration management like user experience with Ansible. User assign roles to hosts/hostgroups and then enforces the policy defined by these roles on a host. Every such Ansible run updates host facts and generates new configuration report. Roles behaviour can be customized by Foreman parametrization that is passed to the Ansible inventory.
+* **Ansible inventory**: Source inventory for Ansible.
+* **Configuration reports**: Inventory of reports from configuration management systems with diff feature and runtime statistics and graphs.
+
+**Compliance management (plugin)**
+
+* **Compliance management**: Define a compliance policy using OpenSCAP standards and tooling, and then enforce it in infrastructure. Supports existing XCCDF profiles and tailoring of them according to user needs.
+
+**Content management (plugin)**
+
+* **Yum and Puppet Repositories**: Create, organize, and manage local yum and puppet repositories. Sync remote repositories or upload content directly to build a library of content that serves as the basis for building custom builds of your content.
+* **Content snapshots**: Take your local content and filter out packages, errata and puppet modules to create custom builds into units called Content Views. Make your custom builds available to your hosts by moving it through environment paths that mimic traditional development workflows (Dev → QE → Stage → Production).
+* **Package and Errata Updates**: Use your locally managed content to install package and errata updates to a host or group of hosts.
+* **Host collections**: A mechanism to statically group multiple Content Hosts. This enables administrators to group Content Hosts based on the needs of their organization. For example, Content Hosts could be grouped by function, department or business unit.
+* **Standard Operating Environment**: Create and maintain a Standard Operating Environment (SOE).
+*
```

Comment:
Please adjust as desired but we could mention following

```
**Reporting capabilities **

* **Report templates **: Thanks to report templates you can generate custom text reports based on data that are available in Foreman. The output can be csv, yaml, json. Templates can contain additional logic and the report can be customized when it's generated.
```

Reply:
Done